demo/meeting/back-end-resources/lambda-layer/nodejs/node_modules/utils.js

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+var AWS = require('aws-sdk');
+var ddb = new AWS.DynamoDB();
+const chime = new AWS.Chime({ region: 'us-east-1' });
+chime.endpoint = new AWS.Endpoint('https://service.chime.aws.amazon.com/console');
+
+const oneDayFromNow = Math.floor(Date.now() / 1000) + 60 * 60 * 24;
+const meetingsTableName = process.env.MEETINGS_TABLE_NAME;
+const attendeesTableName = process.env.ATTENDEES_TABLE_NAME;
+
+// Create a unique id
+exports.uuid = () => {
+  console.log("in uuid")
```

Comment:
- Remove `console.log`s from this file?
- Or fix missing semicolon, single quote